tutorials/event_sampler_tutorial.ipynb

```diff
@@ -0,0 +1,582 @@
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Now, we can finally add the `Skymodel` we want to simulate to the `Dataset` container:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "dataset.models.extend(models_pntpwl)\n",
+    "print(dataset.models)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The next step shows how to sample the events with the `MapdatasetEventSampler` class. The class requests a random number seed generator that we set with `random_state=0`, the `Dataset` and the `Observation` object. From the latter, the `MapDatasetEventSampler` class takes all the meta data information."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "sampler = MapDatasetEventSampler(random_state=0)\n",
+    "events = sampler.run(dataset, observation)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The output of the event-sampler is an event list with coordinates, energies and time of arrivals of the source and background events. Source and background events are flagged by the MC_ID identifier (where 0 is the default identifier for the background)."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "events.table"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "print(f\"Source events: {len(np.where(events.table['MC_ID']==2)[0])}\")\n",
+    "print(f\"Background events: {len(np.where(events.table['MC_ID']==0)[0])}\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We can inspect the properties of the simulated events as follows:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "events.peek()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Let's write the photon event list to a FITS file:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "events.table.write(\"events_0001.fits.gz\", overwrite=True)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Generate a skymap\n",
+    "A skymap of the simulated events is the following:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "counts = Map.create(frame=\"galactic\", skydir=(0, 0.), binsz=0.02, npix=(150, 150))\n",
+    "counts.fill_events(events)\n",
+    "counts.plot(add_cbar=True)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Fit the simulated data\n",
```

Comment:
To keep the tutorial short, perhaps you can just show how to fill `counts.dataset` with the simulated map, and then write the dataset to disc. Explain that at this stage, we have standard reduced datasets which we can fit using standard techniques, eg: `simulate_3d` tutorial.

Reply:
I know this lengthen the tutorial. I was thinking to present also the fitting process in order to show that the observed parameters are compatible with the simulated ones, as an on the fly cross-check. 
However, I can also add the reference to the `simulate_3d` tutorial.